homeassistant/components/gogogate2/__init__.py

```diff
@@ -1 +1,59 @@
 """The gogogate2 component."""
+from homeassistant.components.cover import DOMAIN as COVER_DOMAIN
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+
+from .common import (
+    async_api_info_or_none,
+    create_data_manager,
+    get_api,
+    get_data_manager,
+)
+
+
+async def async_setup(hass: HomeAssistant, base_config: dict) -> bool:
+    """Set up for Gogogate2 controllers."""
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
+    """Do setup of Gogogate2."""
+    # Config entry is coming from config flow and needs options set.
+    if not config_entry.options:
```

Comment:
The current standard is to re-setup the config entry if the authentication credentials change.  We shouldn't import everything into the options.

There is an ongoing discussion about this topic here https://github.com/home-assistant/architecture/issues/377